src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/WorkspaceSteps.java

```diff
@@ -56,39 +56,46 @@ public void setWorkSpacename(final String strname) {
         settingsPage.setName(strname);
     }
 
-    /** This step save workspace. **/
+    /**
+     * This step save workspace.
+     **/
     @And("clicks on Save Button")
-    public void clickSaveButton() {
+    public static void clickSaveButton() {
         final WorkSpaceSettings settingsPage =
                 (WorkSpaceSettings) ScenarioContext.getInstance().getContext(
                         WS_SETTINGS_PAGE);
         settingsPage.clickOnSave();
 
     }
 
-    /** This step display workspace edited. **/
-    @Then("workspace title should be edited")
-    public void getWorkSpaceLabel() {
-        final String name = (String) ScenarioContext.getInstance().getContext(
-                WS_NAME);
-        this.dashboard.goToWorkSpaceTab();
-        assertTrue(this.dashboard.existWorkSpace(name));
-    }
-
-    /** This step create a workspace. **/
+    /**
+     * This step create a workspace.
+     **/
     @Given("create an workspace")
-    public void createaworkspace() {
+    public static void createaworkspace() {
```

Comment:
Why the step definition is static ?

Reply:
I do not know how this come to this, may due to a merge from develop, anyway it should not appear again